### PR TITLE
Add ActiveMQ groovy script

### PR DESCRIPTION
**Pull request by Mrod1598**

### CODE REVIEW

File: activemq.groovy [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def beanBrokerMetrics = otel.mbean("org.apache.activemq:type=Broker,brokerName=*")
+otel.instrument(beanBrokerMetrics, "activemq.totalconnectionscount", "Total connections count per broker", "1", "TotalConnectionsCount", otel.&longSumObserver)
+otel.instrument(beanBrokerMetrics, "activemq.percentageofmemoryusage", "Percentage of configured memory used by the destination", "%", "MemoryPercentUsage", otel.&doubleValueObserver)
+otel.instrument(beanBrokerMetrics, "activemq.currentconnectionscount", "Number of connections that are currently active.", "1", "CurrentConnectionsCount", otel.&longValueObserver)
+otel.instrument(beanBrokerMetrics, "activemq.totalmessages.enqueue", "Total number of messages that have been sent to the broker.", "1", "TotalEnqueueCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:5]: sum?

File: activemq.groovy [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanBrokerMetrics, "activemq.totalmessages.dequeue", "Total number of messages that have been acknowledged from the broker.", "1", "TotalDequeueCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:6]: sum?

File: activemq.groovy [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanBrokerMetrics, "activemq.totalconsumers", "Total number of consumers subscribed to destinations on the broker", "1", "TotalConsumerCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:7]: sum?

File: activemq.groovy [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanBrokerMetrics, "activemq.totalproducers", "Total number of message producers active on destinations on the broker", "1", "TotalProducerCount", otel.&longValueObserver)
+otel.instrument(beanBrokerMetrics, "activemq.totalmessagecount", "Total number of unacknowledged messages on the broker", "1", "TotalMessageCount", otel.&longValueObserver)
+otel.instrument(beanBrokerMetrics, "activemq.averagemessagesize", "Average size of messages in this queue, in bytes", "by", "AverageMessageSize", otel.&longValueObserver)
+
+def beanTopicMetrics = otel.mbean("org.apache.activemq:type=Broker,brokerName=*,destinationType=Topic,destinationName=ActiveMQ.Advisory.MasterBroker")
+otel.instrument(beanTopicMetrics, "activemq.topic.forwardcount", "Number of messages that have been forwarded from this queue to a networked broker", "1", "ForwardCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:13]: is this a sum?

File: activemq.groovy [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanTopicMetrics, "activemq.topic.dequeuecount", " the total number of messages removed from the queue (ack’d by consumer) since last restart", "1", "DequeueCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:14]: is this a sum?

File: activemq.groovy [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanTopicMetrics, "activemq.topic.enqueuecount", "the total number of messages sent to the queue since the last restart", "1", "EnqueueCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:15]: is this a sum?

File: activemq.groovy [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+otel.instrument(beanTopicMetrics, "activemq.topic.consumercount", "The number of consumers subscribed to this topic", "1", "ConsumerCount", otel.&longValueObserver)
+otel.instrument(beanTopicMetrics, "activemq.topic.producercount", "Number of producers publishing to this topic	", "1", "ProducerCount", otel.&longValueObserver)
```
Comment (dehaansa) [inline activemq.groovy:17]: get rid of the extra spaces in the description?

File: activemq.groovy (hunk 2 above)
Comment (Mrod1598) [reply activemq.groovy:6]: [this](https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-activemq.html) is where I saw all the descriptions on whether or not they were counters or gauges

File: activemq.groovy (hunk 7 above)
Comment (Mrod1598) [reply activemq.groovy]: extra spaces are deleted

**Pull request merged**